docs/50-aggregation/5-lookup.mdx

```diff
@@ -148,5 +148,22 @@ The $lookup operation creates an array within each book document. Using $unwind
     ```
      </div>
   </TabItem>
+    <TabItem value="python" label="Python">
+      <div>
+        ```python
+        books_with_reviews = books.aggregate([
+          {
+            "$lookup":
+            {
+              "from": "reviews",
+              "localField": "_id",
+              "foreignField": "_id.bookId",
+              "as": "reviews"
+            }
+          }
+        ])
```

Comment:
@AfiMaameDufie this code is not working correctly. The `foreignField` should be where the book id field is in the `reviews` collection. In this case, it is in `bookId`. Also, we're missing a loop to print out the results. My proposed code is:

```python
books_with_reviews = books.aggregate([
  {
    "$lookup":
    {
      "from": "reviews",
      "localField": "_id",
      "foreignField": "bookId",
      "as": "reviews"
    }
  },
  // use this stage to filter only books that have reviews
  {
    "$match": {
      "reviews.0": { "$exists": True }
    }
  }
])

for book in books_with_reviews:
    title = book.get('title')
    reviews = book.get('reviews')
    print(f"Book Title: {title} - Reviews: {reviews}")
```

Thoughts?